src/Mutation.js

```diff
@@ -81,6 +96,9 @@ function validateFields(data) {
     warning('Ignoring reserved field: updatedAt');
     delete data.updatedAt;
   }
+  if (hasMutlipleGeoPoints()) {
```

Comment:
Rather than declaring a separate function, it seems like you could replace the `return true` with this Error and avoid the `hasMultipleGeoPoints` method entirely.